src/neuroconv/basedatainterface.py

```diff
@@ -75,6 +132,9 @@ def run_conversion(
         nwbfile: Optional[NWBFile] = None,
         metadata: Optional[dict] = None,
         overwrite: bool = False,
+        # TODO: when all H5DataIO prewraps are gone, introduce Zarr safely
+        # backend: Union[Literal["hdf5", "zarr"], HDF5BackendConfiguration, ZarrBackendConfiguration] = "hdf5",
```

Comment:
Including Zarr this early causes problems until we purge all `H5DataIO` usage across our tools, which will occur in one of the next follow-ups, after which Zarr will be re-enabled